Remove printt

prettyprint 0.6 removed printt at the last minute, replace with get_string

Fixes bug 995811

Change-Id: Iaabe47c1ae9270d9d00f804388a1837767cb20f1

--- a/keystoneclient/utils.py
+++ b/keystoneclient/utils.py
@@ -34,14 +34,14 @@
                 row.append(data)
         pt.add_row(row)
 
-    pt.printt(sortby=fields[0])
+    print pt.get_string(sortby=fields[0])
 
 
 def print_dict(d):
     pt = prettytable.PrettyTable(['Property', 'Value'], caching=False)
     pt.aligns = ['l', 'l']
     [pt.add_row(list(r)) for r in d.iteritems()]
-    pt.printt(sortby='Property')
+    print pt.get_string(sortby='Property')
 
 
 def find_resource(manager, name_or_id):
